src/main/java/no/unit/nva/database/UserDb.java

```diff
@@ -0,0 +1,80 @@
+package no.unit.nva.database;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import java.util.Objects;
+
+@DynamoDBTable(tableName = "UsersRoles")
+public class UserDb {
+
+    public static final String ROLE_DELIMITER = ",";
+    public static String TYPE = "User";
+
```

Comment:
Do we want to know the modified/created/etc. of this?

Reply:
Possibly not relevant right here, but we should also consider if changing this results in some cloudwatch event that propagated to CloudTrail so we know how the changes were effected.